settings.h

```diff
@@ -216,3 +216,10 @@ extern SPI_HandleTypeDef SPI_Handle; // Needed to be declared in your main.cpp
 #endif
 
 #endif /* SETTINGS_H */
+
+// See: https://github.com/sparkfun/Arduino_Apollo3/pull/421
+#ifdef AM_PART_APOLLO3
+#ifndef SPI_HAS_TRANSACTION
+#define SPI_HAS_TRANSACTION 1
+#endif
+#endif
```

Comment:
You need to add this before `#endif /* SETTINGS_H */` :)

Reply:
whoops... corrected